src/News/TwitterWatcher.php

```diff
@@ -33,35 +35,35 @@ public function onArticlePublished(NodeProjectedEvent $pbjxEvent): void
         $this->notifyTwitter($event, $node, $pbjx);
     }
 
-    protected function createTwitterNotification(Message $event, Message $article, Pbjx $pbjx): Message
+    protected function createTwitterNotification(Message $event, Message $article, Message $app, Pbjx $pbjx): Message
     {
         $date = $event->get('occurred_at')->toDateTime()->add(new \DateInterval('PT180S'));
         $contentRef = $article->generateNodeRef();
+        $appTitle = $app->get('title');
         $id = UuidIdentifier::fromString(
             Uuid::uuid5(
                 Uuid::uuid5(Uuid::NIL, 'twitter-auto-post'),
-                $contentRef->toString()
+                $contentRef->toString() . $appTitle
             )->toString()
         );
 
         return MessageResolver::resolveCurie('*:notify:node:twitter-notification:v1')::create()
             ->set('_id', $id)
-            ->set('title', $article->get('title'))
+            ->set('title', $article->get('title') . ' | ' . $appTitle)
             ->set('send_at', $date)
             ->set('content_ref', $contentRef);
     }
 
-    protected function getApp(Message $article, Message $event, Pbjx $pbjx): ?Message
+    protected function getApps(Message $article, Message $event, Pbjx $pbjx): ?array
```

Comment:
@meghedi @efjacobson i think you could always return an array here

Reply:
Done.